Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -3870,6 +3870,15 @@ def list_risky_users_or_host_command(client: CoreClient, command: str, args: dic
         ValueError: If the API connection fails.
 
     """
+    def _error_missing_license(e: DemistoException) -> None:
```

Comment:
let's find this method a better name, which hints to future maintainers what happens here without having to read its code.

Reply:
Done